src/Protocols/Pusher/EventHandler.php

```diff
@@ -145,12 +145,12 @@ public function sendInternally(Connection $connection, string $event, array $dat
     /**
      * Format the payload for the given event.
      */
-    public function formatPayload(string $event, array $data = [], ?string $channel = null, string $prefix = 'pusher:'): string|false
+    public function formatPayload(string $event, array $data = [], ?string $channel = null): string|false
```

Comment:
The `$prefix` argument was used only for changing internal events prefix (and in tests). As it seems to have only one purpose - provide `pusher:` prefix, I decided to remove it and provide dedicated implementation for internal events payload.